pkg/storaged/plot.jsx

```diff
@@ -58,8 +58,8 @@ const instances_write_metric = {
 
 export function update_plot_state(ps, client) {
     const devs = [];
-    for (const p in client.drives) {
-        const block = client.drives_block[p];
+    for (const b in client.blocks) {
```

Comment:
This is too much. Let's only include "Drives" and "Other devices".

Reply:
Done.